modules/ROOT/pages/security/ssl-framework.adoc

```diff
@@ -956,6 +956,29 @@ If the certificate is a different path outside of NEO4J_HOME, then set the absol
 dbms.ssl.policy.backup.client_auth=REQUIRE
 ----
 
+=== Configure the backup client for SSL
```

Comment:
bit of a meta comment, but it might be worth including in the documentation, we do not the validate the keys of ssl policy values so
`dbms.ssl.policy.backup.foo` will parse happily but will not help, so it is important that users validate the config keys when configuring SSL, `dbms.ssl.policy.backup.trusted_directory` got me as it must be `dbms.ssl.policy.backup.trusted_dir`